src/ActionBar.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Typography } from "material-ui";
-import { withStyles } from "material-ui/styles";
+import { Typography } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
```

Comment:
are these not both exported from core now?

Reply:
I left them separate to keep things consistent with separating component imports from utility imports. Per their documentation it didn't seem that import withStyles directly from core was supported but it looks like its possible. This would be a quick change for this PR but a size-able one for my changes to perchweb. Do you feel that `withStyles` needs to be imported from core?